rcldotnet/rcldotnet.c

```diff
@@ -46,6 +48,27 @@ void native_rcl_reset_error() { rcl_reset_error(); }
 bool native_rcl_ok() { return rcl_ok(); }
 
 int32_t native_rcl_create_node_handle(void **node_handle, const char *name, const char *namespace) {
+
+  // Validate the node name
+  int validation_result;
+  size_t invalid_index;
```

Comment:
This seems unused.

Reply:
`validation_result` is used, and `invalid_index` has to be declared in order to call `rmw_validate_node_name(name, &validation_result, &invalid_index)`

Reply:
`invalid_index` is not used afterwards though, the documentation for `rmw_validate_node_name` states (http://docs.ros2.org/crystal/api/rmw/validate__node__name_8h.html#af2cd5940838487c2a17b5af96560579e) that you can also pass NULL there if you don't intend to use `invalid_index`

Reply:
Ok, you are right. Fixed.